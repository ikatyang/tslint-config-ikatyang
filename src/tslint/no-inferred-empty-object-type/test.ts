declare function returnGeneric<T>(): T;

// good

const good1 = returnGeneric<any>()

// bad

const bad1 = returnGeneric();
