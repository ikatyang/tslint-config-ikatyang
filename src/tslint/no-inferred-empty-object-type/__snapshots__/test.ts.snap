// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
declare function returnGeneric<T>(): T;

// good

const good1 = returnGeneric<any>()

// bad

const bad1 = returnGeneric();
             ~~~~~~~~~~~~~~~  [Explicit type parameter needs to be provided to the function call]

`;
